include/mt32emu/c_interface/meson.build

```diff
@@ -1,3 +1,6 @@
-foreach file_name, src_file : mt32emu_c_header_map
-  configure_file(input : src_file, output : file_name, copy : true)
+foreach src_file : c_headers
+  file_name = 'string : @0@'.format(src_file).split('/')
+  configure_file(input : src_file,
+                output : file_name[-1],
+                copy   : true)
```

Comment:
One more space before `output` and `copy` is needed to keep the consistent style.

But I think this line is short and readable enough to fit into a single line :)